build.gradle

```diff
@@ -57,6 +57,10 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.google.devtools.ksp' version '1.9.0-1.0.12' apply false
```

Comment:
'1.9.0-1.0.12' is for Kotlin 1.9.0, based on your `version_kotlin` you need '1.8.22-1.0.11' ("$version_kotlin-1.0.11" also works).

Reply:
Thank you very much for checking it out and your hint, I downgraded and I could continue =)